service/controller/v3/resource/chart/desired.go

```diff
@@ -65,3 +73,43 @@ func (r *Resource) getConfigMapValues(ctx context.Context, customObject v1alpha1
 
 	return chartValues, nil
 }
+
+func (r *Resource) getSecretValues(ctx context.Context, customObject v1alpha1.ChartConfig) (map[string]interface{}, error) {
+	secretValues := make(map[string]interface{})
+
+	if key.SecretName(customObject) != "" {
+		secretName := key.SecretName(customObject)
+		secretNamespace := key.SecretNamespace(customObject)
+
+		secret, err := r.k8sClient.CoreV1().Secrets(secretNamespace).Get(secretName, metav1.GetOptions{})
+		if apierrors.IsNotFound(err) {
+			return nil, microerror.Maskf(notFoundError, "secret '%s' in namespace '%s' not found", secretName, secretNamespace)
+		} else if err != nil {
+			return nil, microerror.Mask(err)
+		}
+
+		// TODO: fix this "secret.json" name somewhere and access it in release-operator.
```

Comment:
I think a constant in spec.go would work for this. 

I had a constant in the configmap change but it got removed in the review. It's a good point that the names are used by cluster-operator and release-operator.

Reply:
I thought about putting them into a separate repository, that way operators wouldn't import each other. Question would be if we have something like that already.

Reply:
Ack. Good point on not importing operator packages. We can improve on this later if needed.